hack/make/.build-rpm/docker-engine.spec

```diff
@@ -84,9 +84,12 @@ Requires: device-mapper >= 1.02.90-2
 %if 0%{?fedora} >= 22
 %global selinux_policyver 3.13.1-128
 %endif # fedora 22
-%if 0%{?centos} >= 7 || 0%{?rhel} >= 7 || 0%{?oraclelinux} >= 7
+%if 0%{?centos} >= 7 || 0%{?rhel} >= 7
 %global selinux_policyver 3.13.1-23
-%endif # centos,oraclelinux 7
+%endif # centos,rhel 7
+%if 0%{?oraclelinux} >= 7
+%global selinux_policyver 3.13.1-102.0.3.el7_3.15
```

Comment:
Hm, not sure what is the test case were you hitting, could you please share it ? From your error above, doesn't selinux-policy-targeted were updated to selinux-policy-targeted-3.13.1-102.0.3.el7_3.16.noarch (it just that it is Not found) ? But you are correct, we should also include this dependencies on the docker-engine-selinux.spec file as well, I will update the PR. Thanks.